content/en/docs/setup/independent/high-availability.md

```diff
@@ -250,9 +250,9 @@ done
             # This CIDR is a calico default. Substitute or remove for your CNI provider.
             podSubnet: "192.168.0.0/16"
 
-1.  Replace `x` in `kubernetesVersion: v1.11.x` with the latest available version. 
+1.  Replace `x` in `kubernetesVersion: v1.11.x` with the latest available version.
```

Comment:
v1.12?

Reply:
i spoke with sig-docs about possible means for how to automate such versions with a variable, but the outcome of this discussion was that having such variables in the website framework are not that easy to add.

BTW i think this can be replaced with just `kubernetesVersion: stable`. kubeadm can handle that.
@fabriziopandini @chuckha WDYT?

Reply:
Oh this is for the stacked masters part of the document. I didn't go through those instructions so I didn't modify any of it.

`stable` could work, but it would require some extra thought on the part of the user. For example, the folks testing these instructions would have to remember to change stable to `1.12.0-beta.2` since stable is currently `1.11.3`.

Reply:
if we use `stable` we can have a single comment on top in the lines of
`replace "stable" with the version you want to use - e.g. 1.12.0-beta.2`.

this doesn't require per cycle maintenance.

Reply:
I made this change below, I'm leaving the stacked control planes untouched.

Reply:
> I'm leaving the stacked control planes untouched.

hm, is there a reason for that?
i think we can replace all `v1.11.x` instances with `stable` in the document.